src/olympia/devhub/forms.py

```diff
@@ -1217,12 +1217,24 @@ class DistributionChoiceForm(forms.Form):
         u'updateURL or external application updates.</span>')
 
     channel = forms.ChoiceField(
-        choices=(
-            ('listed', mark_safe_lazy(LISTED_LABEL)),
-            ('unlisted', mark_safe_lazy(UNLISTED_LABEL))),
+        choices=[],
         initial='listed',
         widget=forms.RadioSelect(attrs={'class': 'channel'}))
 
+    def __init__(self, *args, **kwargs):
+        self.addon = kwargs.pop('addon', None)
+        super().__init__(*args, **kwargs)
+        choices = [
+            ('listed', mark_safe_lazy(self.LISTED_LABEL)),
```

Comment:
you won't need the `_lazy` call here because you moved this to ``__init__`` and thus translation happens during runtime.

Reply:
Indeed, thanks. 5beed9ddd20be7badd411385ce052fa8b9c7e197